### ISSUE TITLE
AWS Fargate platform

### ISSUE THREAD

**Issue by tj** [apex/up#449]
Fargate is "container native". This will be a better option for people with more predictable and sustained traffic, where API Gateway can become expensive. 

https://aws.amazon.com/blogs/aws/aws-fargate/

**Comment by huangbong** [comment]
Exciting stuff! Will be great to finally upgrade from Node 6.10.3 and the 50mb code limit in Lambda. I feel like Lambda has always been designed for small scripts between AWS services rather than full fledged Node/Go/Python web servers. Looks like Fargate is the future!

**Comment by tj** [comment]
Agreed, it's a little unfortunate that even regular FaaS with Node is difficult due to the dep sizes, but having more options for different types of apps will be great. Fargate's baseline pricing will be more expensive but it shouldn't be bad

**Comment by sudhirj** [comment]
Also autoscaling.

**Comment by sudhirj** [comment]
Also load balancing - if we're coming into Fargate then the load balancing options - Application vs Network become important.

**Comment by sudhirj** [comment]
Fargate is plain ECS, as far as I can see - it's only a managed cluster on ECS, so it saves having to run instances. Everything above the cluster level is regular ECS, so either load balancer should work. Will double check, though.
